### ISSUE TITLE
SDDM not respecting X11 '-dpi' setting

### ISSUE THREAD

**Issue by johnramsden** [sddm/sddm#943]
I've been experiencing a strange issue where all of my fonts appear squished in plasma 5. I thought it might be an issue with the DPI settings, which I had set to 216 in SDDM with ```ServerArguments=-nolisten tcp -dpi 216```. After attempting to change the DPI value I found that it wasn't actually doing anything. Changing it to 96 and rebooting has no effect versus rebooting with 216.

I currently am using the default X11 config on arch with no configuration file.

I'm not forcing the font DPI in plasma. My fonts used to look fine, so I'm not sure what caused this regression. I'm using plasma 5 on two other computers with the same configuration, and have not experienced the same problem.

**Comment by davidedmundson** [comment]
Run xrdb -q

To see if you are forcing the font DPI

**Comment by johnramsden** [comment]
@davidedmundson so I checked it and it appeared my font _was_ set to a DPI different from the one in SDDM Which made me wonder where else I could be setting it. I then realized I had made some settings in the ```.Xresources``` file. After removing the file, or setting it the same as my X11 DPI the settings seem to be applying again. 

My fonts still appear to be a little squished though, but it only appears to be in plasma, not in the rest of the applications. I'm not sure what might be causing this, but I guess that isn't really SDDM related.

Thanks for helping me with my issue.
